guides/version-control-policies.md

```diff
@@ -0,0 +1,102 @@
+# Version-Control Policies
+
+## TL;DR
+
+- changes which are tightly coupled MUST exist in the same commit.
+- to be accepted into the main branch, commits MUST adhere to any agreed
+  standards.
+- pull-/merge-requests are invariably a workflow on top of your commit history
+  and not the same thing.
+
+## Branches
+
+Typically, we follow the
+[GitHub Flow](https://guides.github.com/introduction/flow/) pattern; we branch
+from, and merge back into, a main branch (by contemporary convention, this is
+usually called `main`.)
+
+Projects MUST define a branching policy.
+
+Projects SHOULD define a convention for branch names. The aim is to convey as
+much information as possible at a glance: this can avoid questions as to the
+intended purpose of changes on a branch.
+
+An example would be the branches created when using Jira's integration with
+version controls systems: this is the
+[_slug_](https://en.wikipedia.org/wiki/Clean_URL#Slug) version of a ticket's
+title. In doing so, it couples the branch to the ticket and conveys as much
+information as is contained in the title.
+
+Projects CAN opt to use traditional prefixes such as `feature/` or `bugfix/`.
+
+## Pull-/Merge-Requests and/or Code Review
+
+We need to make the distinction between the development process and the _merge_
+process.
+
+That is, during development—as per the above—any changes exist on an isolated
+branch, away from the main branch. During this time **_anything goes_**.
+
+Committing frequently, to avoid the possibility of losing either changes or the
```

Comment:
> Worth a separate note about regularly keeping a remote

I reckon so.

Reply:
I'd agree with George on this - pushing is important to not lose changes due to equipment failure/theft/loss/accidents. It's something that's best done regularly instead of periodically. How often should that be? Every 30 mins? Hourly? Twice daily? In other words, what is an acceptable amount of work that you can afford to lose (even if the risk is low)?

I know I'm in the minority with using UI tools for Git, but I'm used to pushing with every commit, it does not make a difference to my workflow or cadence and it gives me confidence that my work is backed-up every time.

Can't comment on terminal users, but probably it won't make a huge difference there either.

Reply:
Updated.